documentation/release-notes/topics/version_5.md

```diff
@@ -1,5 +1,89 @@
 # Version 5
 
+## Hyperview 5.1 (Aug 22, 2025)
+
+This section covers significant changes and bug fixes in Hyperview 5.1.x since version 5.0.x
+
+:::{important}
+1. This release has changes and improvements to the Data Collector software. We strongly recommend updating
+   your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery
+   experience.
+
+2. Customers using Hyperview open-source tools such as hvcli/asset tool must update to the latest version.
+:::
+
+### New Licensed Feature: Accelerated Polling
+
+This feature allows administrators to specify sub five minute polling periods up to once every minute.
+
+:::{note}
+Please contact the Hyperview sales or support teams for more information.
+:::
+
+### Enhanced Feature: 2D and 3D Layout
```

Comment:
We also added color coding by custom property to the location Layout - Map view (google map), Rack View (multi-rack view) and the rack Layout rack elevation.